services/symfony/symfony-insight-base.js

```diff
@@ -87,6 +87,11 @@ class SymfonyInsightBase extends BaseXmlService {
 
   transform({ data }) {
     const lastAnalysis = data.project['last-analysis']
+
+    if (!lastAnalysis) {
+      throw new NotFound({ prettyMessage: 'no analyses found' })
+    }
```

Comment:
Considered addressing this by making `last-analysis` required in the schema, but figured a contextual error message on the badge would be better than the `invalid response data` message especially since we're not sure what sort of projects/scenarios were resulting in all those TypeErrors